packages/optimizer-docker/utils.js

```diff
@@ -1,5 +1,9 @@
 const url = require('url');
 
+function objectFromEntries(arr) {
```

Comment:
why was Object.fromEntries not usable? Would make sense to explain in a comment, because I feel like I would switch this back without seeing it was removed first